scripts/docs/copy_docs.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+version="latest"
+out_dir="./docs_deployment"
+assets_dir="$out_dir/assets"
+index="$out_dir/index.html"
+
+[ -d "$out_dir" ] && rm -r "$out_dir"
+mkdir -p "$out_dir"
+
+# assets
+mkdir -p "$assets_dir"
+cp docs/cosmjs-tree.png "$assets_dir"
+
+{
+  echo '<h1>CosmJS documentation</h1>'
+  echo '<p>CosmJS is a modular library consisting of multiple packages that are organized like this:</p>'
+  echo '<p><a href="assets/cosmjs-tree.png"><img alt="CosmJS dependency graph" src="assets/cosmjs-tree.png" style="max-width: 100%; width: 950px" /></a></p>'
+  echo '<h2>Package documentation</h2>'
+  echo '<p>The following packages are can be used as a library.</p>'
```

Comment:
"can be used as libraries"

Reply:
Faucet ends up in this list.

Reply:
Good point. The faucet could be used as a library as well if there was an index.ts exporting the classes. But I don't see a need for that right now. Will clean up.